editor/src/components/editor/server.ts

```diff
@@ -761,3 +767,24 @@ export function getBranchProjectContents(operationContext: GithubOperationContex
     return response.json()
   }
 }
+
+export async function sendDiscordMessage(
+  webhookType: DiscordWebhookType,
+  messageType: DiscordMessageType,
+  message: DiscordMessage,
+) {
+  const body: DiscordWebhookBody = { webhookType, messageType, message }
```

Comment:
We should be building the message on the back end as much as possible

Reply:
I wanted to keep the endpoint as generic as possible (so we could use it for other channels and notifications as well) - so in the client I only build the "site import" specific information, and the generic things are in the server.
(for example the user id is being added in the backend and not in the client).
do you think there is additional information that should be moved to the server? (note that everything in the client is very specific to the site import process - https://github.com/concrete-utopia/utopia/pull/6662/files#diff-426bff74c9da628f136029668cad91728508ef49a539c96a379c8c1e3f70e150R208-R282)

Reply:
refactored it so that the message is being built on the server:
https://github.com/concrete-utopia/utopia/pull/6662/files#diff-d2102489b75734896e5f1e06678c3f0ca926650eb08cdd14e00e79315f947302R22

and the client just sends what it needs to:
https://github.com/concrete-utopia/utopia/pull/6662/files#diff-d2102489b75734896e5f1e06678c3f0ca926650eb08cdd14e00e79315f947302R22-R80